.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,30 @@
+---
+name: Feature request
+about: Suggest a new feature
+title: ''
+labels: enhancement
+assignees: ''
+
+---
+
+<!--
+Before posting a bug report
+
+Search existing GitHub issues to make sure the issue does not already exist:
+https://github.com/xanaduai/pennylane/issues
+
+For general technical details check out our documentation:
+https://pennylane.readthedocs.io
+-->
+
+#### Feature request description
+
+Description of the feature request - include code snippets or screenshots
+showcasing the requested functionality here if possible. You may use the
+following template.
+
```

Comment:
What about a section for **alternatives** and a section for **references**?

Reply:
Good idea, although I would reckon most users wouldn't necessarily have alternatives or specific references for their feature requests. Perhaps these could just be included in the sections below. I think it's good to keep this as simple as possible, but I'm happy to add more sections if that's beneficial. 🙂